.github/workflows/build.yml

```diff
@@ -219,8 +495,437 @@ jobs:
           GITHUB_REF: ${{ github.ref }}
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
           REPORT_CONTENT: ${{steps.verify_pr.outputs.report_content}}
-          CHART_ENTRY_NAME: ${{ steps.sanity_check_pr_content.outputs.chart-entry-name }}
-          CHART_NAME_WITH_VERSION: ${{ steps.sanity_check_pr_content.outputs.chart-name-with-version }}
+          CHART_ENTRY_NAME: ${{ needs.triage.outputs.chart-entry-name }}
+          CHART_NAME_WITH_VERSION: ${{ needs.triage.outputs.chart-name-with-version }}
+        id: release-charts
+        run: |
+          curl -L -o cr.tar.gz https://github.com/helm/chart-releaser/releases/download/v1.2.0/chart-releaser_1.2.0_linux_amd64.tar.gz
+          tar zxvf cr.tar.gz
+          sudo cp -f cr /usr/local/bin/cr
+          INDEX_BRANCH=$(if [ "${GITHUB_REF}" = "refs/heads/main" ]; then echo "refs/heads/gh-pages"; else echo "${GITHUB_REF}-gh-pages"; fi)
+          CWD=`pwd`
+          cd pr-branch
+          ../ve1/bin/chart-repo-manager --repository=${{ github.repository }} --index-branch=${INDEX_BRANCH} --api-url=${{ github.event.pull_request._links.self.href }} --pr-number=${{ github.event.number }}
+          cd ${CWD}
+
+      - name: Release
+        if: ${{ steps.release-charts.outputs.tag != '' }}
+        uses: softprops/action-gh-release@master
+        continue-on-error: true
+        with:
+          tag_name: ${{ steps.release-charts.outputs.tag }}
+          files: pr-branch/report.yaml
+          fail_on_unmatched_files: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  chart-certification-redhat-community:
```

Comment:
This is not what I was expecting. Single flow for redhat and in the code that generates the index.yaml we change given a flag we set here. The difference between this flow and community is that we continue even when report fails but take into account report to set the charts.openshift.io/providerType: https://github.com/openshift-helm-charts/charts/blob/main/scripts/src/chartrepomanager/chartrepomanager.py#L145
Something like if category == redhat and report_result = failed: providerType = community

Reply:
Originally I tried to add a single `community` job and made it depend on `triage` and `redhat`, then it got skipped everytime when `redhat` job got skipped (in the case where it was a community chart in the first place). I will confirm again because I am fairly certain that something didn't work fundamentally.